README.md

```diff
@@ -34,143 +41,133 @@ pip install "pipecat-ai[option,...]"
 Your project may or may not need these, so they're made available as optional requirements. Here is a list:
 
 - **AI services**: `anthropic`, `azure`, `fal`, `moondream`, `openai`, `playht`, `silero`, `whisper`
-- **Transports**: `daily`, `local`, `websocket`
+- **Transports**: `local`, `websocket`, `daily`, 
 
-## A simple voice agent running locally
+## Code examples
 
-If you’re doing AI-related stuff, you probably have an OpenAI API key.
+There are two directories of examples:
```

Comment:
this sounds a bit weird, since `foundational` is inside `examples`... maybe `There are a bunch of examples ready for you to take a look:`